### ISSUE TITLE
AttributeError when optional argument

### ISSUE THREAD

**Issue by MarekSuchanek** [MarekSuchanek/repocribro#3]
There is an `AttributeError` when optional config argument is used:

```
$ repocribro -c test.cfg db-create
Traceback (most recent call last):
  File "./repocribro/env/bin/repocribro", line 11, in <module>
    load_entry_point('repocribro==0.1.3', 'console_scripts', 'repocribro')()
  File "./repocribro/env/lib64/python3.6/site-packages/repocribro-0.1.3-py3.6.egg/repocribro/manage.py", line 27, in run
    manager.run()
  File "./repocribro/env/lib64/python3.6/site-packages/Flask_Script-2.0.5-py3.6.egg/flask_script/__init__.py", line 412, in run
    result = self.handle(sys.argv[0], sys.argv[1:])
  File "./repocribro/env/lib64/python3.6/site-packages/Flask_Script-2.0.5-py3.6.egg/flask_script/__init__.py", line 353, in handle
    app_namespace, remaining_args = app_parser.parse_known_args(args)
  File "/usr/lib64/python3.6/argparse.py", line 1762, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "./repocribro/env/lib64/python3.6/site-packages/Flask_Script-2.0.5-py3.6.egg/flask_script/__init__.py", line 225, in _parse_known_args
    return self._parse_known_args2(arg_strings, *args, **kw)
  File "/usr/lib64/python3.6/argparse.py", line 1968, in _parse_known_args
    start_index = consume_optional(start_index)
  File "/usr/lib64/python3.6/argparse.py", line 1908, in consume_optional
    take_action(action, args, option_string)
  File "/usr/lib64/python3.6/argparse.py", line 1836, in take_action
    action(self, namespace, argument_values, option_string)
  File "/usr/lib64/python3.6/argparse.py", line 954, in __call__
    items.append(values)
AttributeError: 'str' object has no attribute 'append'

```

**Comment by MarekSuchanek** [comment]
fixed in [9eba994](https://github.com/MarekSuchanek/repocribro/commit/9eba9946fe829cf823954688281b3ca966d774eb)

**Comment by MarekSuchanek** [comment]
fixed in [9eba994](https://github.com/MarekSuchanek/repocribro/commit/9eba9946fe829cf823954688281b3ca966d774eb)
